pkg/provisioningv2/rke2/planner/etcdcreate.go

```diff
@@ -62,39 +54,23 @@ func (p *Planner) runEtcdSnapshotCreate(controlPlane *rkev1.RKEControlPlane, clu
 	return errs
 }
 
-func (p *Planner) generateEtcdSnapshotCreatePlan(controlPlane *rkev1.RKEControlPlane, snapshot *rkev1.ETCDSnapshotCreate, nodeName string) (plan.NodePlan, error) {
+func (p *Planner) generateEtcdSnapshotCreatePlan(controlPlane *rkev1.RKEControlPlane, entry *planEntry) (plan.NodePlan, error) {
 	args := []string{
 		"etcd-snapshot",
 	}
 
-	if snapshot.Name != "" {
```

Comment:
This stuff was removed because it is superfluous to include. `--name` would be the only option we want to specify (and even then, we probably shouldn't allow the user to specify it using the planner to prevent too many permutations of further along the lines)